[2023-11-08T02:42:01.975+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: recalls_etl_v2.clean_raw_data manual__2023-11-08T02:41:55.556062+00:00 [queued]>
[2023-11-08T02:42:01.986+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: recalls_etl_v2.clean_raw_data manual__2023-11-08T02:41:55.556062+00:00 [queued]>
[2023-11-08T02:42:01.986+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-08T02:42:02.004+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): clean_raw_data> on 2023-11-08 02:41:55.556062+00:00
[2023-11-08T02:42:02.011+0000] {standard_task_runner.py:57} INFO - Started process 611 to run task
[2023-11-08T02:42:02.014+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'recalls_etl_v2', 'clean_raw_data', 'manual__2023-11-08T02:41:55.556062+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/recalls_etl_v2.py', '--cfg-path', '/tmp/tmp9bz_62di']
[2023-11-08T02:42:02.015+0000] {standard_task_runner.py:85} INFO - Job 31: Subtask clean_raw_data
[2023-11-08T02:42:02.072+0000] {task_command.py:416} INFO - Running <TaskInstance: recalls_etl_v2.clean_raw_data manual__2023-11-08T02:41:55.556062+00:00 [running]> on host d2f41c2a3488
[2023-11-08T02:42:02.170+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='recalls_etl_v2' AIRFLOW_CTX_TASK_ID='clean_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-08T02:41:55.556062+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-08T02:41:55.556062+00:00'
[2023-11-08T02:42:02.731+0000] {logging_mixin.py:151} INFO - Success: cleaned 29232 recall records.
[2023-11-08T02:42:02.734+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-08T02:42:02.748+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=recalls_etl_v2, task_id=clean_raw_data, execution_date=20231108T024155, start_date=20231108T024201, end_date=20231108T024202
[2023-11-08T02:42:02.789+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-08T02:42:02.822+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
